### PR TITLE
MS-61: Psychologists - CRUD

### PR DESCRIPTION
**Pull request by luisnannini**
*** IMPORTANT NOTES ***
- dayRange and timeRagne validations are commented out, couldnt make them work so you
  can create data by leaving them blank
- update function can´t connect to the backend and doesnt work
- originUrl variable is meant to be applied to every CRUD url

### CODE REVIEW

File: public/js/psychologists.js [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+    timeRange: dataForm[7].value,
+    dayRange: dataForm[8].value,
+  };
+  console.log(dataBody)
+  const url = "http://localhost:4000/api/psychologists/";
+  fetch(url, {
+    method: "POST",
+    body: JSON.stringify(dataBody),
+    headers: {
+      "Content-Type": "psychologist/json",
```
Comment (nicolasblasco) [inline public/js/psychologists.js:89]: @luisnannini content-type doesn't depende of the name of our file. We must write it like this "Content-Type": "application/json".

File: public/js/psychologists.js [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+    },
+  })
+    .then(async (res) => {
+      const data = await res.json();
+      if (res.status === 201) return data;
+      throw new Error(data.msg);
+    })
+    .then((data) => {
+      success(data,"The new psychologist was correctly created:");
```
Comment (nicolasblasco) [inline public/js/psychologists.js:98]: @luisnannini what does this function do? Where is it?

Comment (nicolasblasco) [inline public/js/psychologists.js:98]: @luisnannini We must close the modal in this then

File: public/js/psychologists.js (hunk 1 above)
Comment (luisnannini) [reply public/js/psychologists.js:89]: Lol that explains it failing to connect

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**